lib/sejfguru_web/templates/layout/navbar.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= if assigns[:current_user] do %>
```

Comment:
Whole template is inside `if`. Maybe move `if` into place where template is called to render.